radio/src/targets/pl18/hal.h

```diff
@@ -463,11 +586,15 @@
 // USB
 #define USB_RCC_AHB1Periph_GPIO         RCC_AHB1Periph_GPIOA
 #define USB_GPIO                        GPIOA
-// #define USB_GPIO_VBUS                   GPIO_PIN(GPIOA,9)   // PA.09
 #define USB_GPIO_DM                     GPIO_PIN(GPIOA, 11) // PA.11
 #define USB_GPIO_DP                     GPIO_PIN(GPIOA, 12) // PA.12
 #define USB_GPIO_AF                     GPIO_AF10
 
+#if defined(RADIO_NV14_FAMILY) 
+#define USB_GPIO_VBUS                   GPIO_PIN(GPIOA, 9)  // PA.09
+#define USB_SW_GPIO                     GPIO_PIN(GPIOI, 10) // PI.10
```

Comment:
This is going to be wrong for the NV14, and make the bootloader think it should show the RF USB access menu.

Reply:
These definitions are in nv14 target hal.h, the condition is to prevent from adding to pl18/pl18ev/nb4p

Reply:
It does? When I looked at the schematic for the NV14, nothing is connected to PI.10 🤔

Reply:
Really?  Maybe the schematic that I have is NV14 Pro.  Did our shared drive has this schematic?

Reply:
​Yup, Hardware => Flysky =>FS-NV14
![image](https://github.com/user-attachments/assets/1b1f5cce-6c2c-413c-b910-85e8017e48e8)

Reply:
I figure out why, my version is nv14 v2.2 and renamed to nv14 pro 1.0 and this is the future el18.  The change log state that the usb sw is added in v2.2 and this version is never produced.

Reply:
Pushed a better fix